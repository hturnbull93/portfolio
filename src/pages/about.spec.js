import React from "react"
import { shallow, mount } from "enzyme"
import renderer from "react-test-renderer"
import About from "./about"

describe("About", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<About />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders a Layout", () => {
    const wrapper = shallow(<About />)
    expect(wrapper.find("Layout").length).toEqual(1)
  })

  it("has a h1 containing About", () => {
    const wrapper = shallow(<About />)
    expect(wrapper.find("h1").text()).toEqual("About")
  })
})
