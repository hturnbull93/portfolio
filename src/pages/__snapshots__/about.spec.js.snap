// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches snapshot 1`] = `
<div
  className="layout"
>
  <div
    className="sidebar border"
  >
    <h3>
      <span>
        HT
      </span>
       
      <small>
        Full Stack Developer
      </small>
    </h3>
    <nav>
      <section>
        <h4>
          Sections
        </h4>
        <ul>
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/about"
            >
              About
            </a>
          </li>
        </ul>
      </section>
      <section>
        <h4>
          Links
        </h4>
        <ul>
          <li>
            <a
              href="https://github.com/hturnbull93/cv"
            >
              GitHub CV
            </a>
          </li>
          <li>
            <a
              href="https://twitter.com/hturnbull"
            >
              Twitter
            </a>
          </li>
          <li>
            <a
              href="https://www.linkedin.com/in/harry-turnbull"
            >
              LinkedIn
            </a>
          </li>
          <li>
            <a
              href="https://github.com/hturnbull93"
            >
              GitHub
            </a>
          </li>
          <li>
            <a
              href="https://medium.com/@hturnbull93"
            >
              Medium Blog
            </a>
          </li>
        </ul>
      </section>
    </nav>
  </div>
  <section
    className="content border border-4"
  >
    <h1>
      About
    </h1>
    <p>
      I am a Full Stack developer, looking for new opportunities to work with a team of passionate, smart people, working to build useful apps that consumers enjoy using.
    </p>
    <p>
      I am self-trained and have been working as a freelancer web designer for the past three years. I'm passionate about the intersection of technology and creativity. This led me to train at
       
      <a
        href="https://makers.tech/"
      >
        Makers Academy
      </a>
       (London's leading programming bootcamp), to learn best practices, tools and methodologies, and become a professional Front End Developer.
    </p>
  </section>
  <div
    className="bg"
  />
</div>
`;
